core/src/main/java/org/apache/gravitino/storage/relational/database/H2Database.java

```diff
@@ -54,6 +54,12 @@ public String startH2Database(Config config) {
 
     String connectionUrl = constructH2URI(originalJDBCUrl, storagePath);
 
+    try {
+      Class.forName("org.h2.Driver");
```

Comment:
Why do we need to explicitly load the driver? I remember that the driver-jar is already in the class path.

Reply:
Class.forName(`org.h2.Driver`) is called to load and register the JDBC driver.  
Before JDBC 4.0, Java applications needed to explicitly load the JDBC driver to be used, which was done by calling the `Class.forName()` method and providing the full class name of the driver. 
This call causes the driver class to be loaded into the JVM, and then the driver class's static initializer registers the driver itself with the `java.sql.DriverManager` class so that the DriverManager can use it to establish a database connection.  Without this call, when you try to establish a database connection via `DriverManager.getConnection()`, the DriverManager may not find a suitable driver to connect to the database and therefore throws an exception.

Reply:
I am confused about why it worked before without needing an explicit call?

Reply:
+1, this may be caused by shade of integration test common

Reply:
I used `mergeServiceFiles()` to fix this problem.